TestMatrix.md

```diff
@@ -80,6 +81,22 @@ The switch does not support the MTU on a per-interface basis. It does not suppor
 
 * mtu
 
+### network_interface
+
+#### 2960
+
+#### 4507
+
+#### 6503
+
+The above devices do not support snmp trap mac-notification.
```

Comment:
It would be better to refer to them as the params `mac_notification_added` and `mac_notification_removed`, same with `ip_dhcp_snooping_trust` and `_limit`

Reply:
have changed the wording